apis/core/lib/domain/identifiers.ts

```diff
@@ -55,3 +56,12 @@ export function user(user: string): NamedNode {
 export function job(jobCollection: GraphPointer<NamedNode>): NamedNode {
   return $rdf.namedNode(`${jobCollection.value}/${nanoid()}`)
 }
+
+export function dimensionMetadata(dimensionMetadataCollection: DimensionMetadataCollection, mapping: ColumnMapping): NamedNode {
+  if (mapping.targetProperty.termType === 'Literal') {
+    return $rdf.namedNode(`${dimensionMetadataCollection.id.value}/${mapping.targetProperty.value}`)
+  } else {
+    const prefixedProperty = shrink(mapping.targetProperty.value) || mapping.targetProperty.value
+    return $rdf.namedNode(`${dimensionMetadataCollection.id.value}/${prefixedProperty}`)
+  }
+}
```

Comment:
1. May be slightly more readable if there was only one `return`?
2. Please `urlencode` the URI if did not shrink